images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -243,4 +243,8 @@ function Get-AptPackages {
     $apt = $toolsetJson.apt
     $pkgs = ($apt.common_packages + $apt.cmd_packages | Sort-Object) -join ", "
     return $pkgs
+}
+
+function Get-PipxVersion {
+    return "Pipx $(pipx --version 2> $null)"
```

Comment:
I am not sure `2> $null` will work. You can use `Get-CommandResult` from SoftwareReport.Helpers

Reply:
The point is that `Get-CommandResult` reroutes stderr to stdout, but `pipx --version` writes some unnecessary information that looks like `25l25h` to stderr. So, I've decided rerouting stderr to null is the simplest way to resolve it.